<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminDialog</class>
 <widget class="QDialog" name="AdminDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>253</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>456</width>
    <height>253</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>253</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>仪器管理员面板</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../controller.qrc">
    <normaloff>:/images/controller.ico</normaloff>:/images/controller.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size: 20px;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>439</width>
     <height>215</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>11</number>
    </property>
    <item row="0" column="0">
     <widget class="QToolButton" name="trainingButton">
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>96</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Training</string>
      </property>
      <property name="icon">
       <iconset resource="../controller.qrc">
        <normaloff>:/images/btn_training.png</normaloff>:/images/btn_training.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="maintainButton">
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>96</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Maintain</string>
      </property>
      <property name="icon">
       <iconset resource="../controller.qrc">
        <normaloff>:/images/btn_maintain.png</normaloff>:/images/btn_maintain.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QToolButton" name="assistButton">
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>96</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>By Admin</string>
      </property>
      <property name="icon">
       <iconset resource="../controller.qrc">
        <normaloff>:/images/btn_assist.png</normaloff>:/images/btn_assist.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="educationButton">
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>96</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Teaching</string>
      </property>
      <property name="icon">
       <iconset resource="../controller.qrc">
        <normaloff>:/images/education.png</normaloff>:/images/education.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="shutdownButton">
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>96</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>End</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/images/btn_shutdown.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="recordButton">
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>96</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Record</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/images/btn_table.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QToolButton" name="settingsButton">
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>96</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Setting</string>
      </property>
      <property name="icon">
       <iconset resource="../controller.qrc">
        <normaloff>:/images/btn_settings.png</normaloff>:/images/btn_settings.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="projectStartButton">
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>96</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Project</string>
      </property>
      <property name="icon">
       <iconset resource="../controller.qrc">
        <normaloff>:/images/project.png</normaloff>:/images/project.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="userInfoGroupBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>12</y>
     <width>150</width>
     <height>220</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>220</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>225</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string>Active User</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>132</width>
      <height>195</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="userPhotoLabel">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>161</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>180</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../controller.qrc">:/images/user_default.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="userInfoLabel">
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../controller.qrc"/>
 </resources>
 <connections/>
</ui>
